conformance/lang/expressions/multiplicative-expr/multiplicative_expr.balt

```diff
@@ -6,9 +6,6 @@ function errorFunction(int a, float b) {
     int _ = a * b; // @error * is not allowed with operands of different basic types
```

Comment:
This is allowed now. The error here is the result type is `float`. Let's check the other similar places as well

Reply:
Updated the test in b890cb7.